### DISCUSSION TITLE
Pre-Crash Systems:

### DISCUSSION THREAD

**Discussion by SinisterSticks** [darthcloud/BlueRetro#722] [Ideas]
I'm willing to fund either some seed money or pay in advance for physical copies of Bluetooth to adapters for the following systems:

Intellivision, ColecoVision, Atari 5200, Bally astrocade, Vectrex.

The first three systems use an extra keypad for 12 digit input but have to be wired in such a way because there's not 12 independent inputs for each of the 12 buttons,  but some use multi pin combinations others use some other thing I don't understand.

I'll put down $25 in research money and front $25 towards the first one you build of such things.

**Comment by darthcloud** [comment]
Sorry, you probably mean well,  but do you realize how ridiculous it is to offer pennies for a month+ worth of engineering effort.

I do BlueRetro because I love doing so. Out of blue donation are welcome and makes me happy.

But this kind of bounty doesn't help anything getting done.
There is like easily 100K$ worth of engineering time I did on BlueRetro in the last 4 years.

**Comment by SinisterSticks** [comment]
I'm sorry.    I understood my contribution alone probably wouldn't be enough.    I wasn't suggesting it would be worth it if    I alone would be the only person offering the bounty.
  

  
I was thinking if other potential users of other consoles offered a similar bounty of $25 for seed research plus a $25 advance on the physical device you would get enough that would be worth for you doing it.
  

  
I don't know how easy it is to convert to other devices.    Some things are easy to repeat over and over and just change slightly.    Other things require radically different changes.    I was hoping it would be an easy enough conversion where a bunch of people combining $50 contributions might make it work.
  

  
Now I'm assuming you're on open source which means you welcome other people to help you and you're welcome to help other people.
  

  
If that's the case then maybe I could facilitate a hookup between you and a couple of people that are addressing the pre-crash systems.
  

  
There's Edladdin.com who deals in Colecovision and Atari 7800.    And there's Retrogameboyz.com who deals with Atari 5200 and Intellivision.
  

  
I can put down $25 as kind of an extra thank you for me buying a couple devices that weren't directly approved by you but used your tech.    There's a couple of people on eBay using your technology and making pre-built versions of stuff that supposedly people who can build them should be able to build. (There was someone who builds a four pack of N64 adapters for 30 bucks and a similar four pack for GameCube adapters for around $30.)
  

  

  
If I have a vote on where my $25 goes,    it would be towards pre-crash systems.    I understand I am ine vote among many.    (Not as many as a Brook gaming device, but small enough to be one among many) Of course this is kind of fluid.    If there's not enough movement for pre crash systems, use it however you feel.    Don't feel like it's trapped in the box of pre-crash systems only.    If I'm outvoted, hey, I'm outvoted.
  

  
I just didn't know if technology was easily translatable to stuff like these pre-crash systems.
  

  
Now looking back on it, those who do pre-crash systems usually don't do the post-crash systems and vice versa.
  

  
I'll see if Edladdin or Retrogameboys wants to exchange data with you.    They might give you info on pre crash stuff.    Since you do mainly modern bluetooth, and I assume if your project is open source, they are welcome to work on pre-crash versions of your tech if the bluetooth aspects are properly acknowledged and licensed.    I noticed RetroOnyx is both where I bought my Virtual Boy BT adapter, and listed as a featured friend specializing in Virtual Boy in your list of credits.
  

  
If you wish, accept a $25 donation on July 1st.    No strings attached.    Mainly because I benefited from your tech, and as just one vote among many.    Plus I'll see if Edladdin and RetroGameBoyz are interested in working on pre-crash versions of your Bluetooth tech, assuming you allow it, which is a fairly safe assumption. But if not, tell me now.     
  

  

  
  
  
  
>

**Reply by SinisterSticks** [reply]
Hello

I'm  just wondering if there's been any more action on the pre crash stuff? Any more donations that are loosely attributed to pre crash Bluetooth adapters?

**Reply by SinisterSticks** [reply]
Hello.  I just remembered that an AtariAge user "SadGirlRecords" have a small quantities of 2600 controller to Odyssey 2 adapters and 2600 to Bally Astrocade adapters (for joystick and fire button only,  paddle not carried over)

In addition to Edladdin for 7800 qnd Colecovision and RetroGameBoyz for 5200 and Intellivision, maybe you can consult these entities and work out something to add to the DB9 and DB15 adapters.

(I never used the term parallel for those type of adapters.  I think more people who are on the console side are familiar with the terms DB9 or "9 pin" [most pre-crash systems, 7800, Sega Master, and Genesis]  and DB1 or "15 pin" [5200, Famicom, Neo Geo] )
